ca_gazebo/models/camera/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+    <name>camera</name>
```

Comment:
Use a better name. Why are you using the camera?

Reply:
Changed name to april_tags_camera